Lint python code (PEP8 standard)

Lint following files:
- [x] plugins.d/python.d.plugin
- [x] python.d/python_modules/base.py
- [x] python.d/python_modules/base/*.py
- [x] python.d/[a-c]*.py
- [x] python.d/[d-h]*.py
- [x] python.d/[i-l]*.py
- [x] python.d/m*.py
- [x] python.d/[n-o]*.py
- [x] python.d/[p-r]*.py
- [x] python.d/[s-z]*.py

Current status:
```
python.d/web_log.chart.py:505:32: E128 continuation line under-indented for visual indent
python.d/memcached.chart.py:9:1: E265 block comment should start with '# '
python.d/postgres.chart.py:93:121: E501 line too long (134 > 120 characters)
python.d/postgres.chart.py:188:121: E501 line too long (140 > 120 characters)
python.d/postgres.chart.py:189:121: E501 line too long (142 > 120 characters)
python.d/postgres.chart.py:190:121: E501 line too long (142 > 120 characters)
python.d/postgres.chart.py:191:121: E501 line too long (143 > 120 characters)
python.d/postgres.chart.py:236:121: E501 line too long (142 > 120 characters)
python.d/postgres.chart.py:250:121: E501 line too long (140 > 120 characters)
python.d/postgres.chart.py:295:121: E501 line too long (121 > 120 characters)
python.d/postgres.chart.py:361:121: E501 line too long (136 > 120 characters)
python.d/postgres.chart.py:366:121: E501 line too long (136 > 120 characters)
python.d/postgres.chart.py:376:121: E501 line too long (135 > 120 characters)
python.d/fail2ban.chart.py:29:21: E131 continuation line unaligned for hanging indent
python.d/beanstalk.chart.py:118:5: E124 closing bracket does not match visual indentation
python.d/python_modules/__init__.py:1:1: W391 blank line at end of file
python.d/python_modules/__init__.py:1:1: W293 blank line contains whitespace
```

Seems like we only need those to be fixed and we could make PEP8 as a hard requirement:
```
python.d/postgres.chart.py:93:121: E501 line too long (134 > 120 characters)
python.d/postgres.chart.py:188:121: E501 line too long (140 > 120 characters)
python.d/postgres.chart.py:189:121: E501 line too long (142 > 120 characters)
python.d/postgres.chart.py:190:121: E501 line too long (142 > 120 characters)
python.d/postgres.chart.py:191:121: E501 line too long (143 > 120 characters)
python.d/postgres.chart.py:236:121: E501 line too long (142 > 120 characters)
python.d/postgres.chart.py:250:121: E501 line too long (140 > 120 characters)
python.d/postgres.chart.py:295:121: E501 line too long (121 > 120 characters)
python.d/postgres.chart.py:361:121: E501 line too long (136 > 120 characters)
python.d/postgres.chart.py:366:121: E501 line too long (136 > 120 characters)
python.d/postgres.chart.py:376:121: E501 line too long (135 > 120 characters)
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -21,7 +21,7 @@ jobs:
     - stage: "lint"
       name: "python"
       install: skip
-      script: docker run -ti --rm -v $(pwd):/apps alpine/flake8:3.5.0 --max-line-length=120 --exclude=third_party,urllib3,pyyaml3,pyyaml2 python.d/ plugins.d/python.d.plugin || echo "OK"
+      script: docker run -ti --rm -v $(pwd):/apps alpine/flake8:3.5.0 --max-line-length=120 --exclude=third_party,urllib3,pyyaml3,pyyaml2 python.d/ plugins.d/python.d.plugin
     - name: "javascript"
       install: skip
       script: docker run -it --rm -v $(pwd)/web:/code eeacms/jslint --color /code/web/*.js /code/plugins.d/node.d.plugin/*.js /code/node.d/*.js /code/node.d/node_modules/netdata.js

